packages/matchbox/src/components/Tag/Tag.js

```diff
@@ -43,7 +47,10 @@ class Tag extends Component {
     const closeMarkup = onRemove
       ? <UnstyledLink
         className={styles.Close}
-        onClick={onRemove}>
+        onClick={onRemove}
+        onKeyDown={(e) => this.handleKeydown(e, onRemove)}
```

Comment:
We have keydown helpers in `helpers/keyEvents`. Could slim some of this down.

```js
onKeyDown={onKey('space', props.onRemove)}
```

Reply:
Ooh cool. I’ll definitely check those out and clean it up.

Reply:
Alrighty, swapped it out. Nice API there!